validator/validator-main.protoascii

```diff
@@ -3523,6 +3507,7 @@ tags: {
   }
   attr_lists: "input-common-attr"
   attr_lists: "input-name-attr"
+  attrs: {name: "[type]"}
```

Comment:
Consider adding a comment above that this binding is not allowed on all formats.